README.md

```diff
@@ -151,7 +151,9 @@ Here are the following definitions of each variable in the above example:
 <a name="base64-encode-problematic-parameters"></a>
 ## Base64 encode problematic parameters
 
-When dealing with complex inputs, encoding can be difficult to deal with and implement. To help with this, very parameter in imgix has a Base64 alias, which allows the values to be encoded using the "base64url" encoding with URL and filename safe alphabet ([RFC 4648](https://en.wikipedia.org/wiki/Base64#RFC_4648)). These parameters are keyed by appending `64` to the end of the parameter name. Thus `txt` becomes `txt64`.
+When dealing with complex inputs, encoding can be difficult to deal with and implement. To help with this, every parameter in imgix has a Base64 alias, which allows the values to be encoded using the "base64url" encoding with URL and filename safe alphabet ([RFC 4648](https://en.wikipedia.org/wiki/Base64#RFC_4648)). These parameters are keyed by appending `64` to the end of the parameter name. Thus `txt` becomes `txt64`.
+
+It is extremely important to note that traditional Base64 encoding and Base64 urlsafe encoding are not interchangeable.  Encoding parameter values as regular Base64 WILL cause your requests to fail.
```

Comment:
I think we may want to more clearly note that there are lots of base64's (not just normal vs. urlsafe), and only the RFC 4648 compliant urlsafe one will fly.